app/src/main/java/com/orangeelephant/sobriety/ui/common/MillisecondsToDate.kt

```diff
@@ -0,0 +1,14 @@
+package com.orangeelephant.sobriety.ui.common
```

Comment:
This probably belongs in a util folder rather than UI since its not a composable